src/app/devices/views/Devices.tsx

```diff
@@ -1,30 +0,0 @@
-import { Route, Switch } from "react-router-dom";
-
-import DeviceDetails from "./DeviceDetails";
-import DeviceList from "./DeviceList";
-
-import NotFound from "app/base/views/NotFound";
-import devicesURLs from "app/devices/urls";
-
-const Devices = (): JSX.Element => {
-  return (
-    <Switch>
-      <Route
-        exact
-        path={devicesURLs.devices.index}
-        render={() => <DeviceList />}
-      />
-      {[
-        devicesURLs.device.configuration(null, true),
-        devicesURLs.device.index(null, true),
-        devicesURLs.device.network(null, true),
-        devicesURLs.device.summary(null, true),
```

Comment:
We'd no longer need to define all the subsections like this as they're captured by the top level `/devices/*` path.